app/schemas/models/user.coffee

```diff
@@ -189,6 +189,7 @@ _.extend UserSchema.properties,
   purchased: c.RewardSchema 'purchased with gems or money'
   deleted: {type: 'boolean'}
   dateDeleted: c.date()
+  doNotDeleteEU: {type: 'boolean'}
```

Comment:
This should be a date so that we can use this same mechanism if they go another long period of inactivity and we have to ask them again.